code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm

```diff
@@ -36,6 +36,16 @@
 	var/list/gorilla_overlays[GORILLA_TOTAL_LAYERS]
 	var/oogas = 0
 
+	search_objects = 1
+	wanted_objects = list(/obj/item/reagent_containers/food/snacks/grown/banana)
```

Comment:
if someone else wants to suggest some items a gorilla would want, please comment them here

Reply:
https://seaworld.org/animal-info/animal-infobooks/gorilla/diet-and-eating-habits/